sharktank_models/action.yml

```diff
@@ -19,21 +19,39 @@ inputs:
     description: "Type of target to test"
   GPU:
     description: "Type of GPU to test"
+  BACKEND:
+    description: "Type of backend to test"
   EXTERNAL_TEST_FILE_DIRECTORY:
     description: "The directory of test files and configurations"
   QUALITY_TEST_DIRECTORY:
     description: "The directory of quality tests"
   BENCHMARK_TEST_DIRECTORY:
     description: "The directory of benchmark tests"
+  IREE_TEST_FILES:
+    description: "Directory of the iree test files"
 
 runs:
   using: "composite"
   steps:
-    - name: Checkout repository 
+    - name: Checkout IREE repository
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2 
       with:
         clean: false # doesn't clean up the venv created before this composite action
```

Comment:
This is error prone - always assuming that this action will be run from the IREE repository.

What does this action actually need from that repository? Can those files be passed in to this workflow directly instead?